include/directx/d3d12.h

```diff
@@ -2300,7 +2302,8 @@ enum D3D_SHADER_MODEL
         D3D_SHADER_MODEL_6_4	= 0x64,
         D3D_SHADER_MODEL_6_5	= 0x65,
         D3D_SHADER_MODEL_6_6	= 0x66,
-        D3D_SHADER_MODEL_6_7	= 0x67
+        D3D_SHADER_MODEL_6_7	= 0x67,
+        D3D_HIGHEST_SHADER_MODEL = D3D_SHADER_MODEL_6_7 // TODO: Pull from latest d3d12.h

```

Comment:
Added a "HIGHEST_VERSION" entry to help identify the highest version. Note: I find this entry to be present in the internal D3D12 library header but not in the open source repo. Probably updated recently?

Reply:
Good find!  This is indeed a recent addition that will make it across into the public headers the next time we update it.